### PR TITLE
cmd/snappy, snappy: remove use of snappy.Part in cmd/snappy

### PR DESCRIPTION
**Pull request by mvo5**
[based on https://github.com/ubuntu-core/snappy/pull/651]

Remove the usage of snappy.Part from the "cmd/snappy" part of the code.

### CODE REVIEW

File: snappy/snap_remote_repo.go [hunk 1]
```diff
@@ -345,7 +345,21 @@ func (s *SnapUbuntuStoreRepository) Search(searchTerm string) (SharedNames, erro
...
+		parts[i] = snap
+	}
+	return parts, nil
+}
+
+// SnapUpdates returns the available updates as RemoteSnap types
+func (s *SnapUbuntuStoreRepository) SnapUpdates() (snaps []*RemoteSnap, err error) {
 	// the store only supports apps, gadget and frameworks currently, so no
 	// sense in sending it our ubuntu-core snap
 	//
```
Comment (chipaca) [inline snappy/snap_remote_repo.go]: I suspect this comment is now less than accurate.

**Comment by chipaca** [comment]
Do we still need the methods on `SnapLocalRepo` that return `[]Part`? Here's hoping they too can die.

:+1:

**Comment by mvo5** [comment]
retest this please

**Comment by mvo5** [comment]
@chipaca the []Part bits will go in one of the follow up branches.

**Comment by mvo5** [comment]
retest this please

**Pull request merged**